domain/src/Plugin/EntityReferenceSelection/DomainAdminSelection.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\domain\Plugin\EntityReferenceSelection;
+
+use Drupal\domain\Plugin\EntityReferenceSelection\DomainSelection
+
+/**
+ * Provides entity reference selections for the domain entity type.
+ *
+ * @EntityReferenceSelection(
+ *   id = "default:domain_admin",
+ *   label = @Translation("Domain administrator selection"),
+ *   entity_types = {"domain"},
+ *   group = "default",
+ *   weight = 1
+ * )
+ */
+class DomainSelection extends DomainSelectionSelection {
```

Comment:
Should this not be `class DomainAdminSelection extends DomainSelection`?
There are no traces of `DomainSelectionSelection` other than here